### PR TITLE
D9 number guessing game

### PR DESCRIPTION
**Pull request by karan8404**
This approach seems more visualizable. Inform me if I should instead stick to the question(and do that on top of this approach or scrap this approach altogether).
P.S. - It can be easily solved with 4 guesses most times.. sometimes a 5th attempt is required and i have never encountered a case when i lost(if i play perfectly)

### CODE REVIEW

**Comment by Gadzrux** [comment]
I would say stick to the format in the question

File: A/D9/number_guessing_game.c
Comment (Gadzrux) [inline A/D9/number_guessing_game.c:1]: when the numbers are in the correct position, the numbers don't show as correct anymore.
for example:
Generating a Random Number!...
Random Number Genarated!...
You have 5 Attempts to find the number.

Enter your guess[A 4 digit number]
1234
2 numbers are correct and 0 numbers are in correct position
Result:[x->not present , *->present , _->proper position]
1 2 3 4
* x x *

Incorrect Guess...4 Attempts left!

Enter your guess[A 4 digit number]
1456
3 numbers are correct and 0 numbers are in correct position
Result:[x->not present , *->present , _->proper position]
1 4 5 6
* * x *

Incorrect Guess...3 Attempts left!

Enter your guess[A 4 digit number]
4167
0 numbers are correct and 3 numbers are in correct position
Result:[x->not present , *->present , _->proper position]
4 1 6 7
_ _ _ x

Incorrect Guess...2 Attempts left!

Enter your guess[A 4 digit number]
4168
0 numbers are correct and 3 numbers are in correct position
Result:[x->not present , *->present , _->proper position]
4 1 6 8
_ _ _ x

Incorrect Guess...1 Attempts left!

Enter your guess[A 4 digit number]
4169
0 numbers are correct and 3 numbers are in correct position
Result:[x->not present , *->present , _->proper position]
4 1 6 9
_ _ _ x

Incorrect Guess...0 Attempts left!
Run out of attempts.. You Lost!. The number was : 4160

sorry for the format issue, md files have different meaning for characters like star.The entire code is working fine just work on the problem mentioned above

File: A/D10/merge_sorted.c
Comment (Gadzrux) [inline A/D10/merge_sorted.c:1]: a very elegant solution, loved it.

**Review by Gadzrux — CHANGES_REQUESTED** [review]

**Comment by Gadzrux** [comment]
resolve conflicts

File: A/D9/number_guessing_game.c
Comment (karan8404) [reply A/D9/number_guessing_game.c:1]: That was intentional but i will change it anyway if it's better this was

**Pull request closed without merging**